page_templates/style_guide.md

```diff
@@ -0,0 +1,73 @@
+
+4.  When explaining general concepts, paragraphs should be used. Yet paragraphs should not be over six sentences long.
+
+5.  Use the second person tense and describe actions to the user. (this is the same as active style or imperative style)
+
+> You should open Jina using python.
+> In your pod, you will find.
+
+6.  When using Jina terms, they should be Capitalized. Flow, Pod, JinaD. Anything found in the Jina glossary or Jina 101. Including Executor types (Evaluator, Segmenter, etc.)
+
+7. Don't capitalize Flow types (index, query). Don't capitalize the adjective or  `-tion`  form (i.e. evaluation, evaluating, segmenting, etc)
+
+8.  Do not use compound sentences, chains of clauses, and location-specific idioms, making text hard to understand and translate.
+
+9.  Avoid semicolons.
+
+10.  Don’t use abbreviations without explaining them first. With the expection of abbreviations and acronyms that are common in the computer industry. Examples include YAML, API, GIF, PDF, REST, ID. If you are unsure if you should define something or not, please ask in Slack.
+
+11. When discussing a Jina component inside a code context (i.e. referring to CLI output or actual Python/YAML code) use  Flow,  Pod, etc
+
+12. For code/CLI output, capitalization should match the actual code or output
+
+	    #correct:
+        import numpy as np
+    	from jina.executors.encoders import BaseImageEncoder
+
+    	#incorrect
+    	import Numpy as NP
+    	from jina.executors.encoders import BaseImageEncoder
+
+13.  - For top-level titles: Use Title Case [You can check with this website [here\]
+](https://titlecaseconverter.com/)		- For section titles: Use sentence case
+
+14. For code markup:
+- Use `<code>` in HTML or `` ` `` in Markdown to apply a monospace font and other styling to [code in text](https://developers.google.com/style/code-in-text), inline code, and user input.
+- Use code blocks, `<pre>` or ` ``` `, for [code samples](https://developers.google.com/style/code-samples) or other blocks of code.
+- Do not override or modify font styles inline.
+- Use code font to mark up code, such as class names, method names, HTTP status codes, console output, and placeholder variables.
+
+15. If you need to refer to installing Jina, use the following snippet:
+
+| Jina Product | Snippet to use |
+|--|--|
+| Jina Core | **Requirements**: You have installed the latest stable release of Jina Core according to the instructions found [here](https://docs.jina.ai/chapters/install/index.html).  |
+| Jina Dashboard | **Requirements**: You have installed the latest stable release of Jina Dashboard according to the instructions found [here](https://github.com/jina-ai/dashboard).  |
+| Jina Box | **Requirements**: You have installed the latest stable release of Jina Box according to the instructions found [here](https://docs.jina.ai/chapters/box/introduction/index.html).  |
```

Comment:
Moving forwards perhaps we should make a collection of snippets in a central location. Not urgent